eng/Packages.Data.props

```diff
@@ -161,7 +161,7 @@
   -->
   <ItemGroup>
     <PackageReference Update="Microsoft.Azure.AutoRest.CSharp" Version="3.0.0-beta.20230104.3" PrivateAssets="All" />
-    <PackageReference Update="Azure.ClientSdk.Analyzers" Version="0.1.1-dev.20221031.1" PrivateAssets="All" />
+    <PackageReference Update="Azure.ClientSdk.Analyzers" Version="0.1.1-dev.20221229.1" PrivateAssets="All" />
```

Comment:
Since this updates the central location everyone will from now on get the new package, is there any process we follow to ensure no-one else breaks with it? It is likely not to be the case and there should not be breaking change but curious.

Reply:
I would have expected this PR to kick off the same pipelines as Core for that reason, but I don't see that happening.  @weshaggard, do you know if there is a way to check?

Reply:
This pipeline should kick of the core pipeline which is a proxy for all the other pipelines as it does build those other projects that depend on core. Right now looks like there is a merge conflict which is why I expect the pipelines aren't triggering.

Reply:
Got it - thanks, @weshaggard!  I'll resolve that, and we'll go from there.  Thanks!